File: app/src/main/java/com/example/megaport/mynews/Models/Articles.java
package com.example.megaport.mynews.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

// Class used to manipulate a list of applications

public class Articles {

    @SerializedName(value="results", alternate={"docs"})
    @Expose
    private List<Article> articles = null;

    public List<Article> getArticles() { return articles; }

    public Article getArticle(int position){ return articles.get(position);}

    public void setArticles(List<Article> articles) {
        this.articles = articles;
    }

    @SerializedName("status")
    private String status;

    public String getStatus() {
        return status;
    }

}

File: app/src/main/java/com/example/megaport/mynews/models/Articles.kt
package com.example.megaport.mynews.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

// Class used to manipulate a list of applications

class Articles {

    @SerializedName(value = "results", alternate = ["docs"])
    @Expose
    var articles: List<Article>? = null

    @SerializedName("status")
    val status: String? = null

    fun getArticle(position: Int): Article? {
        return articles?.get(position)
    }

}
